aait/backend/group-2B/Application/DTOs/Notifications/NotificationListDto.cs

```diff
@@ -0,0 +1,9 @@
+
+namespace SocialSync.Application.DTOs.Notifications;
+public class NotificationListDto : INotificationDto
+{
+    public int SenderId { get; set; }
+    public int RecepientId { get; set; }
+    public NotificationType NotificationType{get; set;}
```

Comment:
make the type of NotificationType string

Reply:
Done